src/test/java/gdsc/konkuk/platformcore/controller/member/MemberControllerTest.java

```diff
@@ -130,4 +135,36 @@ void should_fail_when_existingMember() throws Exception {
 			.andExpect(status().isBadRequest())
 			.andDo(print());
 	}
+
+	@Test
+	@DisplayName("회원 탈퇴 성공")
+	@CustomMockUser
+	void should_success_when_delete_member() throws Exception {
+		//given
```

Comment:
`@WithMockUser` 대신 `@CustomMockUser`을 사용하는 이유가 있나요?

Reply:
WithMockUser에는 기본적으로 
UserDetails의 기본적인 항목들이 들어갑니다.
```
String value() default "user";

	/**
	 * The username to be used. Note that {@link #value()} is a synonym for
	 * {@link #username()}, but if {@link #username()} is specified it will take
	 * precedence.
	 * @return
	 */
	String username() default "";

	/**
	 * <p>
	 * The roles to use. The default is "USER". A {@link GrantedAuthority} will be created
	 * for each value within roles. Each value in roles will automatically be prefixed
	 * with "ROLE_". For example, the default will result in "ROLE_USER" being used.
	 * </p>
	 * <p>
	 * If {@link #authorities()} is specified this property cannot be changed from the
	 * default.
	 * </p>
	 * @return
	 */
	String[] roles() default { "USER" };

	/**
	 * <p>
	 * The authorities to use. A {@link GrantedAuthority} will be created for each value.
	 * </p>
	 *
	 * <p>
	 * If this property is specified then {@link #roles()} is not used. This differs from
	 * {@link #roles()} in that it does not prefix the values passed in automatically.
	 * </p>
	 * @return
	 */
	String[] authorities() default {};

	/**
	 * The password to be used. The default is "password".
	 * @return
	 */
	String password() default "password";

```
저희 코드에서는 UserDetail을 커스텀하여 새로 만들어 사용하고 있는데요. 추후에 만들어 나갈 기능에서 커스텀한 유저 디테일에서 Username, authority만 가져오는게 아니라 비활성화라던가 여러가지 부가적인 필드가 필요할 수있다고 생각해서 CustomMockUser을 만들었습니다. :)